### PR TITLE
Context menu in native transactions screen

### PR DESCRIPTION
**Pull request by nicoglennon**
Add avatar context menu in native transaction screen. Fixes RNBW-1291.

### CODE REVIEW

**Comment by nicoglennon** [comment]
Pending: Discover page `firstResponder` issue

**Comment by nicoglennon** [comment]
https://www.loom.com/share/7ef34ae42a524b7394663a4c7465e9a4

**Review by skylarbarrera — APPROVED** [review]
🍳

**Comment by jinchung** [comment]
Pending Michal response: unmount vs mount of Discover sheet when presenting / dismissing (hack for showing the context menu)

File: src/hooks/useOnAvatarPress.ts [hunk 1]
```diff
@@ -71,7 +93,7 @@ export default () => {
           : undefined,
         options: avatarActionSheetOptions,
       },
-      async buttonIndex => {
+      async (buttonIndex: Number) => {
         if (buttonIndex === 0) {
           ImagePicker.openPicker({
```
Comment (jinchung) [inline src/hooks/useOnAvatarPress.ts:98]: Can this just be `onAvatarChooseImage`

File: src/hooks/useOnAvatarPress.ts [hunk 2]
```diff
@@ -85,20 +107,54 @@ export default () => {
             });
```
Comment (jinchung) [inline src/hooks/useOnAvatarPress.ts:107]: can this `navigate` be replaced by `onAvatarPickEmoji`

**Comment by jinchung** [comment]
> Pending Michal response: unmount vs mount of Discover sheet when presenting / dismissing (hack for showing the context menu)

Given the changes to the 300 API calls, this is less of a concern!

**Pull request merged**